ScheduleTask.DAL/Interfaces/TaskRepository.cs

```diff
@@ -1,27 +1,30 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Data.Entity;
+﻿using System.Collections.Generic;
 using System.Linq;
-using System.Web;
-using Task1.Models;
+using System.Runtime.Serialization.Formatters;
+using ScheduleTask.DAL.Entities;
 
-namespace Task1.Services
+namespace ScheduleTask.DAL.Interfaces
 {
 
-    public interface ITaskService
+    public interface ITaskRepository
     {
         Tasks GetById(int id);
         List<Tasks> GetTasks();
         void AddTask(Tasks task);
```

Comment:
Why do wee need Word Task here? Why can`t we have base repository with all these actions like add/Get/getbyId/update/delete?